ts/features/itwallet/discovery/screens/ItwIpzsPrivacyScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import { View } from "react-native";
+import { H2, IOStyles, VSpacer } from "@pagopa/io-app-design-system";
+import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel";
+import I18n from "../../../../i18n";
+import LoadingSpinnerOverlay from "../../../../components/LoadingSpinnerOverlay";
+import ItwPrivacyWebViewComponent from "../components/ItwPrivacyWebViewComponent";
+import { ItwEidIssuanceMachineContext } from "../../machine/provider";
+import { trackOpenItwTosAccepted } from "../../analytics";
+
+const IPZS_PRIVACY_URL = "http://io.italia.it/informativa-ipzs";
```

Comment:
I would move this URL to the environment file.

Reply:
Why would you move this to an environment file? Do you think it might change between a development build or a production one?

Reply:
Usually I am not a fan of hardcoded urls in the code. I'm not sure if it can change between dev and prod, but it might change in the future, and I find it easier to go straight to the env file rather than looking for it in the code.

Besides, if you look at `io-app`'s env file you will find other privacy policy urls.

Reply:
Thanks, @gispada, I moved the URL inside the `env` file.